### PR TITLE
Initial Query Trips Implementation

### PR DESCRIPTION
**Pull request by zghera**
### What is a quick description of the change?

Initial implementation to query and populate all trips associated with a user on the view trips page.

### Is this fixing an issue?

None.

### Are there more details that are relevant?

The trips that are queried were created manually in the database. Additionally, the user email that was used to query for trips was obtained from a hard-coded getuserEmail() function that will be implemented in the Authentication portion of the MVP. Also for this PR, the only details of the trip that are passed to the UI is the trip name for simplicity sake. A future PR will feed all of the trip details to the UI and make them look pretty (Issue #17).

### Check lists (check `x` in `[ ]` of list items)

- [  ] Test written/updating
- [  ] Tests passing
- [x] Coding style (indentation, etc)

**Still need to add tests if possible.**

### Any additional comments?

{Fill in here or remove (leave question)}

### CODE REVIEW

File: slurp/src/main/webapp/scripts/view-trips.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+// limitations under the License.
+
+
+/**
+ * Fetches the previously entered comments from the server and inserts each
+ * comment as a list item of the 'comments-thread-container' <ul> element.
+ *
+ * The number of comments displayed is determined by
+ * getNumCommentstoDisplay().
+ */
```
Comment (keiffer01) [inline slurp/src/main/webapp/scripts/view-trips.js:22]: Looks like this still references comments stuff.

File: slurp/src/main/webapp/scripts/view-trips.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      .get()
+      .then(querySnapshot => {
+        querySnapshot.forEach(doc => {
+          console.log(typeof doc.data());
+          console.log(doc.data());
+          tripsContainer.appendChild(createTripElement(doc.id));
+        });
+      })
+      .catch(error => {
+        console.log('Error in getCommentsThread: ' + error);
```
Comment (keiffer01) [inline slurp/src/main/webapp/scripts/view-trips.js:37]: As above.

File: slurp/src/main/webapp/scripts/view-trips.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+ * and landmark latitude-longitude coordinates. However, both the blobKey
+ * and landmark fields of the comment JSON object can be null. If the
+ * blobKey value null, no image element is included in the parent <li> element.
+ * If the landmark value is null, no landmark information is included.
+ *
+ * @param {object} commentInJson A string that contains a JSON object of an
+ *    individual comment. This JSON object contains fields for message text,
+ *    image, landmark name, and landmark latitude-longitude coordinates.
+ * @return {HTMLDivElement} The div element containing the trip data.
+ */
```
Comment (keiffer01) [inline slurp/src/main/webapp/scripts/view-trips.js:60]: This one too haha.

File: slurp/src/main/webapp/scripts/view-trips.js (hunk 1 above)
Comment (zghera) [reply slurp/src/main/webapp/scripts/view-trips.js]: Fixed documentation in 84de185.

File: slurp/src/main/webapp/scripts/view-trips.js (hunk 2 above)
Comment (zghera) [reply slurp/src/main/webapp/scripts/view-trips.js]: 84de185

File: slurp/src/main/webapp/scripts/view-trips.js (hunk 3 above)
Comment (zghera) [reply slurp/src/main/webapp/scripts/view-trips.js]: 84de185

**Pull request closed without merging**